listenbrainz/webserver/views/api.py

```diff
@@ -76,6 +76,9 @@ def submit_listen():
     if mb_engine and current_app.config["REJECT_LISTENS_WITHOUT_USER_EMAIL"] and not user["email"]:
         raise APIUnauthorized(REJECT_LISTENS_WITHOUT_EMAIL_ERROR)
 
+    if user['is_paused']:
+        raise APIUnauthorized("user_id is paused and is currently not accepting listens. Feel free to Contact Us if you have any questions about this.")
```

Comment:
I can see no reason why "Contact Us" should be uppercase here in the middle of a sentence. Please lowercase it.

Should this also include a link to the Contact Us page?

Would it make sense to have a constant for this in `utils.py` like for `REJECT_LISTENS_WITHOUT_EMAIL_ERROR` just above? Similarly, does this need to also be implemented separately for last.fm compatibility in https://github.com/metabrainz/listenbrainz-server/blob/d9ef6f3d5916709363946ee61258c8ab1746da29/listenbrainz/webserver/errors.py#L68 ?

Reply:
Yes, lets fix the case and also add a link to the contact us page. But, I am confused about the comment about the constant.... And yes, compat API also needs to be implemented! I will post another comment for this issue.

Reply:
Well, the previous error in this page at https://github.com/metabrainz/listenbrainz-server/blob/89684cd853537fa983fb0b04c36f1aea2a7909d8/listenbrainz/webserver/views/api.py#L77 uses a constant for the error text. If this needs to be reused for compat api, then probably double the reason to have a constant rather than reusing the string :)